test/unit/puppetlabs/services/jruby/jruby_service_test.clj

```diff
@@ -31,7 +62,7 @@
        (try
          (bootstrap/with-app-with-config
           app
-          jruby-testutils/default-services
+          (conj jruby-testutils/default-services jruby-pooled-test-service)
```

Comment:
not sure I understand why adding the new service to this particular test is necessary?